Standards/scs-XXXX-vN-key-manager-standard.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Key-Manager Standard
+type: Standard
+status: Draft
+track: IaaS
+---
+
+## Introduction
+
+To encrypt user data like volumes or in the future also Images and ephemeral storage for VMs, the key is needed to be present in the infrastructure.
+Therefore a key-manager is needed to store the keys and apply authorization policies on requests.
+The OpenStack key-manager implemenation that can bu utilized for this is named Barbican.
+This standard aims to provide a base level of security for Cloud Service Providers that integrate a key-manager into their deployments.
+
+## Terminology
+
+| Term | Meaning |
+|---|---|
+| API | Application Programming Interface, often referring to the REST API interfaces provided by OpenStack and related services |
+| CSP | Cloud Service Provider, provider managing the OpenStack infrastructure |
+| IaaS | Infrastructure-as-a-Service |
+| HSM | Hardware Security Module |
+| KEK | Key Encryption Key |
```

Comment:
I suggest to add Barbican into the Terminology to reduce repeating in the spec itself.